### DISCUSSION TITLE
MIB2 showing both Bose and Burmester options.  how to remove one?

### DISCUSSION THREAD

**Discussion by vartech125** [Mr-MIBonk/M.I.B._More-Incredible-Bash#179] [Q&A]
Currenlty everything works great just seeing both burmester and bose as options . I have mine using optical out so if possible I would like to keep the Burmester option as long as it does not stop my optical output from working

**Comment by harman-f** [comment]
where exactly do you see this?

**Reply by vartech125** [reply]
I see it on main screen at main there is a Burmester displayed at the bottom center and I can select it and change settings.  Also under my screen I see the Bose logo and I can select it and change options.

**Reply by harman-f** [reply]
not that familiar with that car. Pictures would help.
Patch does not change any car coding or basic functions.

**Reply by vartech125** [reply]
![24445D59-40D0-4109-A90D-23AD476902F7](https://user-images.githubusercontent.com/116739535/232367398-90bc5185-250c-43e7-a79e-b7a3651963d1.jpeg)
![EA1EB038-CB10-457B-B71F-1CB897E3BC33](https://user-images.githubusercontent.com/116739535/232367402-d4d8597f-5950-4458-9e6d-af66dcc694b6.jpeg)
Here are the pictures of the screens. I want in to make changes and they both are no longer available. They disappeared.

**Reply by harman-f** [reply]
Please share your full story!
This is a retofit?

**Reply by vartech125** [reply]
Ok. I have a 981S cayman that had a pcm3.1 with Bose. I decided to retrofit the PCM 4.0 from a 718 with Bose.   I ran the mib tool installed gem and ran the all in one patch. Audio is working via optical cable. The weird thing is Burmester appeard on the radio along with Bose. When I tried to select some of th options available under Bose or Burmester menu it would not let me. When I went back to the main menu they were gone. This is all under the pcm normal menu. I would is it possible to and one of them back. Preferably the Burmester since I have 10 channels similar to Burmester on the aftermarket amp and interface. Let me know if this helps and thanks again for the responses.

**Reply by harman-f** [reply]
Do you have any sound at all?

This is way out of the scope from M.I.B ;)
PCM 4.0 unit will have 718 coding on it.
Are there any 981S cayman which came with PCM4 from factory?
If so, you would need to get datasets from such a car.
Get a M.I.B backup from that car and share it here.

Datasets have to fit the hardware and coding most likely has to be adjusted.

**Reply by vartech125** [reply]
I do have sound. 981s never came with pcm4. Will sharing a backup from the pcm4 that I have installed help? If so how do I do that?  I want to get back one of the two options Bose or Burmester preferably Burmester and one addition thing active backup camera can that be done in gem?

**Comment by harman-f** [comment]
You should try to get datasets from the 3.1 unit. They are mostly compatable with MHI2. 

You could use carscanner to get them.

**Reply by vartech125** [reply]
would take a lot of work to remove the pcm after all the rewiring.  not worth it.
